components/overflow-group/overflow-group.js

```diff
@@ -15,7 +15,7 @@ import { LocalizeCoreElement } from '../../lang/localize-core-element.js';
 import { offscreenStyles } from '../offscreen/offscreen.js';
 import ResizeObserver from 'resize-observer-polyfill/dist/ResizeObserver.es.js';
 import { RtlMixin } from '../../mixins/rtl-mixin.js';
-import { throttle } from 'lodash-es';
+import throttle from 'lodash-es/throttle';
```

Comment:
Interesting. So before was it important all the things?

Reply:
Yeah not sure how I didn't notice this. For the tree shaking to work properly you have to import the individual function file, once I started testing in the lms it became obvious